src/go/k8s/config/crd/kustomization.yaml

```diff
@@ -1,29 +1,47 @@
 # This kustomization.yaml is not intended to be run by itself,
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+#+kubebuilder:scaffold:crdkustomizeresource
 resources:
 - bases/redpanda.vectorized.io_clusters.yaml
 - bases/redpanda.vectorized.io_consoles.yaml
 - bases/cluster.redpanda.com_redpandas.yaml
 - bases/cluster.redpanda.com_topics.yaml
-#+kubebuilder:scaffold:crdkustomizeresource
 - bases/toolkit.fluxcd.io/helm-controller.yaml
 - bases/toolkit.fluxcd.io/source-controller.yaml
 
-patches:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
-- path: patches/webhook_in_clusters.yaml
-- path: patches/webhook_in_redpanda_consoles.yaml
-- path: patches/webhook_in_cluster.redpanda.com_topics.yaml
```

Comment:
NIT: This movement break the kubebuilder (I think). If in the future you would like to add new webhook configuraiton it will end up here (before `+kubebuilder:scaffold:crdkustomizewebhookpatch` line).

For anyone that would look at this file it would be hard to understand if patches where added manually or by kubebuilder.